doc/check.texi

```diff
@@ -1301,11 +1302,12 @@ srunner_add_suite (sr, make_pack_suite ());
 @end verbatim
 @end example
 
-@node Selective Running of Tests, Testing Signal Handling and Exit Values, Multiple Suites in one SRunner, Advanced Features
+@node Selective Running of Tests, Selecting Tests Based on Arbitrary Tags, Multiple Suites in one SRunner, Advanced Features
```

Comment:
Makes sense. Slight delay while I figure out how to do subsections .. but I'll do that.